src/Microsoft.Data.SqlClient/src/Microsoft/Data/ProviderBase/DbConnectionPool.cs

```diff
@@ -453,10 +459,7 @@ internal DbConnectionPool(
             _pooledDbAuthenticationContexts = new ConcurrentDictionary<DbConnectionPoolAuthenticationContextKey, DbConnectionPoolAuthenticationContext>(concurrencyLevel: 4 * Environment.ProcessorCount /* default value in ConcurrentDictionary*/,
                                                                                                                                                         capacity: 2);
 
-            if (ADP.s_isPlatformNT5)
-            {
-                _transactedConnectionPool = new TransactedConnectionPool(this);
-            }
+            _transactedConnectionPool = new TransactedConnectionPool(this);
```

Comment:
I guess this is pretty much given nowadays, right?

Reply:
Exactly, yes - every supported version of .NET Framework / .NET Core requires NT 5.0+.